modules/price/src/Plugin/Field/FieldFormatter/PriceCalculatedFormatter.php

```diff
@@ -84,11 +84,7 @@ public static function create(ContainerInterface $container, array $configuratio
    * {@inheritdoc}
    */
   public function viewElements(FieldItemListInterface $items, $langcode) {
-    $currency_codes = [];
-    foreach ($items as $delta => $item) {
-      $currency_codes[] = $item->currency_code;
-    }
-    $currencies = $this->currencyStorage->loadMultiple($currency_codes);
+    $currencies = $this->currencyStorage->loadMultiple();
```

Comment:
I agree with Matt. We also need a test for multicurrency, to demonstrate that the use case is working with the fix.